rtb/norme-di-progetto/src/introduzione.tex

```diff
@@ -2,14 +2,55 @@ \section{Introduzione e scopo del documento}
 
     \subsection{Scopo del documento}
 
+    Il presente documento si pone lo scopo di individuare e definire le best practices$^{G}$ e il Way of Working$^{G}$ del progetto che ogni componente del gruppo SWEet16
+    si impegna a rispettare durante l’intero svolgimento del progetto Easy Meal. \\
+    In questo modo si cercherà di garantire omogeneità e coesione in ogni aspetto del suddetto progetto.
+  
     \subsection{Scopo del prodotto}
 
+    Lo scopo dell’applicazione è quello di creare una piattaforma che permetta di gestire e semplificare il processo di prenotazione$^{G}$ di tavoli all’interno dei ristoranti. \\
+    Sarà inoltre possibile anticipare l’esperienza culinaria visionando prima il menù ed andando ad effettuare la propria ordinazione$^{G}$ prima di arrivare al ristorante. \\
+    Il prodotto offre inoltre un’esperienza di ordinazione delle pietanze$^{G}$ collaborativa e coinvolgente, permettendo di condividerla con amici ed, in caso di dubbi, interagire direttamente con lo staff del ristorante.
+
+    L’idea è una piattaforma SaaS$^{G}$ (Software as a Service), in cui i saranno presenti due tipi di utenti:
+    \begin{itemize}
+        \item Clienti$^{G}$: utente registrato all’interno dell’applicazione, può cercare ristoranti, effettuare prenotazioni, ordinazioni e inserire feedback e recensioni;
+        \item Ristoratori$^{G}$: utente registrato all’interno dell’applicazione, può gestire uno o più ristoranti, controllando le prenotazioni e le ordinazioni dei clienti ed i menù del/i ristorante/i.
+    \end{itemize}
+
+
+    La piattaforma dovrà essere  disponibile attraverso una WebApp$^{G}$ accessibile da qualsiasi dispositivo oppure tramite dispositivo mobile, attraverso dispositivo iOS$^{G}$ oppure Android$^{G}$.
```

Comment:
dispositivo mobile sottointende ios o android, perciò non è ridondante ?

Reply:
Non ero sicuro di questa parte in effetti
Ora che lo rileggo risulta ridondante